README.md

```diff
@@ -91,7 +91,7 @@ Every voice is important. If you have something on your mind, create an issue or
 
 ## Included content
 
-<!-- Galaxy will eventually list the module docs within the UI, but until that is ready, you may need to either describe your plugins etc here, or point to an external docsite to cover that information. -->
+<!-- Galaxy now displays full module docs within the UI. If you don't use Galaxy for your collection, you may need to either describe your plugins etc here, or point to an external docsite to cover that information. -->
```

Comment:
There's still the glaring bug in Galaxy that doesn't show any documentation uses docs fragments from dependent collections. Also it shows plugin docs as well (but no role argument spec docs). How about:
```suggestion
<!-- Galaxy now usually displays full module and plugin docs within the UI. If you don't use Galaxy for your collection, you may need to either describe your plugins etc here, or point to an external docsite to cover that information. -->
```